asgraphite.py

```diff
@@ -653,67 +658,102 @@ def run(self):
                         msg.extend(lines)
                 except Exception as e:
                     print("Unable to parse set stats:")
-                    print(r)
+                    print(res)
                     print(e)
                     sys.stdout.flush()
 
             if args.latency:
-                r = -1
+                res = -1
```

Comment:
It will be good if you can put an example string which we are parsing here. It's hard to verify the parsing logic without knowing the format of string.

Reply:
Are you talking about an example for latencies_cmds or for r/res?  Here is an example of r if one just runs with cmd `latencies:`:
```
batch-index:;{test}-read:;{test}-write:;{test}-udf:;{test}-query:;{bar}-read:;{bar}-write:;{bar}-udf:;{bar}-query:
```
here is r with cmd `latencies:hist={test}-benchmark-write` with write benchmarks enables:
```
{test}-write-start:;{test}-write-restart:;{test}-write-dup-res:;{test}-write-master:;{test}-write-repl-write:;{test}-write-response:
```

Are you asking for me to add a comment with this info in it?

Reply:
Yes. It would be good to have this comment in code, where it is been parsed. It will be easy for anyone to look at comment and know what is being parsed and in what format.

Reply:
You got it!